x-pack/plugins/osquery/public/timelines/add_to_timeline_button.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { isArray } from 'lodash';
+import { EuiButtonEmpty, EuiButtonIcon } from '@elastic/eui';
+import { useKibana } from '../common/lib/kibana';
+
+const TimelineComponent = React.memo((props) => <EuiButtonEmpty {...props} size="xs" />);
+TimelineComponent.displayName = 'TimelineComponent';
+
+export interface AddToTimelineButtonProps {
+  field: string;
+  value: string | string[];
+  isIcon?: true;
+  iconProps?: Record<string, string>;
+}
+
+export const SECURITY_APP_NAME = 'Security';
+export const AddToTimelineButton = (props: AddToTimelineButtonProps) => {
+  const { timelines, appName } = useKibana().services;
+  const { field, value, isIcon, iconProps } = props;
+
+  const queryIds = isArray(value) ? value : [value];
+  const TimelineIconComponent = useCallback(
+    (timelineComponentProps) => (
+      <EuiButtonIcon iconType={'timelines'} {...timelineComponentProps} size="xs" {...iconProps} />
```

Comment:
you don't need the `{}` around `"timelines"`

Reply:
Thank Michael! I'll change the {} in another PR, thanks for catching this 👍